primary/src/metrics.rs

```diff
@@ -69,7 +69,7 @@ pub struct PrimaryChannelMetrics {
     pub tx_primary_messages: IntGauge,
     /// occupancy of the channel from the `primary::PrimaryReceiverHandler` to the `primary::Helper`
     pub tx_helper_requests: IntGauge,
-    /// occupancy of the channel from the `primary::ConsensusAPIGrpc` to the `primary::BlockWaiter`
+    /// occupancy of the channel from the `primary::ConsensusAPIGrpc` & `executor::Subscriber` to the `primary::BlockWaiter`
```

Comment:
I'd add one line saying that in contexts with an external consensus, it will be the former, and when the consensus is internal, it will be the later. This should make clear we don't actually expect 2 simultaneous senders yet.